readme.md

```diff
@@ -9,150 +9,125 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**remark**][remark] plugin to inject Git contributors into a markdown table.
-Collects contributors from Git history, deduplicates them, augments it with
-metadata found in options, a module, or `package.json` and calls
-[`remark-contributors`][contributors] to render the markdown table.
-
-## Note!
-
-This plugin is ready for the new parser in remark
-([`micromark`](https://github.com/micromark/micromark),
-see [`remarkjs/remark#536`](https://github.com/remarkjs/remark/pull/536)).
-No change is needed: it works exactly the same now as it did before!
-(note that `remark-gfm` must be used as well to support tables)
+**[remark][]** plugin to generate a list of Git contributors.
 
 ## Contents
 
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
 *   [Install](#install)
 *   [Use](#use)
-    *   [Inject](#inject)
-    *   [Metadata](#metadata)
-*   [Supported properties](#supported-properties)
 *   [API](#api)
     *   [`unified().use(remarkGitContributors[, options])`](#unifieduseremarkgitcontributors-options)
+*   [Examples](#examples)
+    *   [Example: CLI](#example-cli)
+    *   [Example: CLI in npm scripts](#example-cli-in-npm-scripts)
+    *   [Example: `appendIfMissing`](#example-appendifmissing)
+    *   [Example: metadata](#example-metadata)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
 *   [Security](#security)
 *   [Contribute](#contribute)
 *   [Contributors](#contributors)
 *   [License](#license)
 
-## Install
+## What is this?
+
+This package is a [unified][] ([remark][]) plugin that collects contributors
+from Git history, deduplicates them, augments it with metadata found in options,
+a module, or `package.json`, and passes that to
+[`remark-contributors`][remark-contributors] to add them in a table in
+`## Contributors`.
+
+**unified** is a project that transforms content with abstract syntax trees
+(ASTs).
+**remark** adds support for markdown to unified.
+**mdast** is the markdown AST that remark uses.
+This is a remark plugin that transforms mdast.
 
-This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c):
-Node 12+ is needed to use it and it must be `import`ed instead of `require`d.
+## When should I use this?
+
+This project is particularly useful when you have (open source) projects that
+are maintained with Git and want to show who helped build them by adding their
+names, websites, and perhaps some more info, based on their commits, to readmes.
+This package is useful because it’s automated based on Git: those who commit
+will get included.
+The downside is that commits aren’t the only way to contribute
```

Comment:
What would you want to mention about it?

Reply:
This comment is specifically tied to
> The downside is that commits aren’t the only way to contribute

All contributors aims to address this by including other contribution types.
It could also go in a related project section or some such, or omitted entirely.

Just thought it was worth mentioning given the caveat.